scripts/release/debian/test_deb_package.py

```diff
@@ -14,7 +14,7 @@
 pytest_parallel_cmd = '{} -n auto'.format(pytest_base_cmd)
 
 for mod_name in mod_list:
-    if mod_name in ['botservice', 'network']:
+    if mod_name in ['botservice', 'network', 'cloud']:
```

Comment:
What's the purpose of this line

Reply:
Add cloud to the modules that need to be run in serial.